### ISSUE TITLE
Protocol grid flickers every time an electrode is clicked (Trac #130)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#134]
This probably indicates that the entire grid is being redrawn in this case, which shouldn't be necessary.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/130
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "This probably indicates that the entire grid is being redrawn in this case, which shouldn't be necessary.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Protocol grid flickers every time an electrode is clicked", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-10-30T19:49:02", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20121119T22:42:41: 
* ryan commented:

> commit 5c1a98fca08c0a20c6d234ca0c6c856475130bc1
> Author: Ryan Fobel <ryan@fobel.net>
> Date: 2012-11-19T16:29:20Z-0500
> 
>     Fixes #134
> 
>     Don't redraw protocol grid for on_protocol_changed event (only need
>     to redraw for on_protocol_swapped or on_step_options_changed).


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "" to "microdrop"
